modules/charon-core/src/main/java/org/wso2/charon3/core/protocol/endpoints/BulkResourceManager.java

```diff
@@ -80,6 +83,24 @@ public SCIMResponse processBulkData(String data, UserManager userManager) {
             bulkRequestProcessor.setFailOnError(bulkRequestDataObject.getFailOnErrors());
             bulkRequestProcessor.setUserManager(userManager);
 
+            int maxOperationCount =
+                    (Integer) CharonConfiguration.getInstance().getConfig().get(SCIMConfigConstants.MAX_OPERATIONS);
+            int totalOperationCount = bulkRequestDataObject.getUserOperationRequests().size() +
+                    bulkRequestDataObject.getGroupOperationRequests().size() +
+                    bulkRequestDataObject.getRoleOperationRequests().size() +
+                    bulkRequestDataObject.getRoleV2OperationRequests().size();
+            if (totalOperationCount > maxOperationCount) {
+                if (logger.isDebugEnabled()) {
+                    logger.debug(String.format(ResponseCodeConstants.ERROR_DESC_MAX_OPERATIONS_EXCEEDED,
+                            totalOperationCount,
+                            maxOperationCount));
```

Comment:
```suggestion
                            totalOperationCount,
                            maxOperationCount));
```
Should fit in one line

Reply:
Addressed in https://github.com/wso2/charon/pull/408.